### PR TITLE
Faucet account creation

### PR DESCRIPTION
**Pull request by kasunkavinda**
Account creation

### CODE REVIEW

File: src/pages/TestnetFaucet/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    }
+  }
+  
+  return (
+    <Fragment>
+      <Button variant="contained" className='mb-4' disabled = {accountButtonClick} onClick={() => testnetFaucet()}>Generate an account and process the fund transaction</Button>      
+      {(accountButtonClick) ? <Loader className="p-4" /> : null}
+      {(accountGeneratedFlag) ? <FaucetBox faucetBox = {faucetBox}/> : null}
+      {warningDisplay && <Alert severity="warning">Too Fast!! Try again!</Alert>}
```
Comment (kithminisg) [inline src/pages/TestnetFaucet/index.js:39]: Try again shortly.

File: src/services/Evernode.js [hunk 2]
```diff
@@ -119,4 +121,84 @@ const onLedger = async (callback) => {
...
+                        // Clearing the interval after success
+                        clearInterval(faucetInterval);
+                    } catch (error) {
+                        resolve(FaucetAccount.faucetAccountCreationError);
+
+                        // Clearing the interval after failure to avoid error loop
+                        clearInterval(faucetInterval);
+                    }
+                    
+                }, 5000)
```
Comment (chalith) [inline src/services/Evernode.js:159]: Do we need this?

File: src/services/Evernode.js [hunk 3]
```diff
@@ -119,4 +121,84 @@ const onLedger = async (callback) => {
...
+            }).then(async()=> {
+                const tenantClient = new evernode.HostClient(new_wallet.address, new_wallet.seed);
```
Comment (chalith) [inline src/services/Evernode.js:161]: client?

File: src/services/Evernode.js [hunk 4]
```diff
@@ -119,4 +121,71 @@ const onLedger = async (callback) => {
...
+                await hostClient.xrplAcc.makePayment(hostClient.config.foundationAddress,
+                    evernode.XrplConstants.MIN_XRP_AMOUNT,
+                    evernode.XrplConstants.XRP,
+                    null,
+                    [{ type: 'giftBetaEvr', format: '', data: '' }]);
+
+                // Keep watching our EVR balance.
+                let attempts = 0;
+                while (attempts >= 0) {
+                    await new Promise(resolve => setTimeout(resolve, 1000));
```
Comment (chalith) [inline src/services/Evernode.js:164]: Wouldn't this be ambiguous since the parent resolve function is also in the same name

**Review by chalith — APPROVED** [review]

**Review by kithminisg — APPROVED** [review]

**Pull request merged**